service/service.go

```diff
@@ -71,22 +71,30 @@ func NewService(options Options, registry *plugin.Registry) *Service {
 
 func (s *Service) Start() error {
 	if s.options.Log == "console" {
+		log.SetOutput(os.Stdout)
 		log.SetFormatter(&log.TextFormatter{})
 	} else if s.options.Log == "syslog" {
-		hook, err := logrus_syslog.NewSyslogHook("", "", syslog.LOG_INFO, "")
+		devNull, err := os.OpenFile("/dev/null", os.O_WRONLY, 0)
+		if err != nil {
+			panic(err)
```

Comment:
this argument makes sense, but in practice that means that all new containers with vulcand inside will crash, so we have no option here but to fallback to stdout to preserve backwards compatibility.